pkg/module/kubebuilder/project.go

```diff
@@ -144,30 +145,53 @@ func (p *Project) DefaultCR(s step.Step) ([]byte, error) {
 	}
 
 	samplesDir := filepath.Join(p.path, samplesPath)
-	d, err := os.ReadDir(samplesDir)
+	filez, err := os.ReadDir(samplesDir)
 	if err != nil {
 		return nil, fmt.Errorf("could not read samples dir %q: %w", samplesDir, err)
 	}
 
-	if len(d) == 0 {
+	if len(filez) == 0 {
 		return nil, fmt.Errorf("no default CR available: samples directory %q is empty", samplesDir)
 	}
 	defaultCR := ""
-	if len(d) > 1 {
+	if len(filez) > 1 {
 		// ask for specific file
-		names := []string{}
-		for _, f := range d {
-			names = append(names, f.Name())
+		var promptString strings.Builder
+		promptString.WriteString(fmt.Sprintf("Please specify the file to use as default CR in %s:\n", samplesDir))
+
+		files := map[int]string{}
+		for index, file := range filez {
+			shiftedIndex := index + 1
+			files[shiftedIndex] = file.Name()
+			promptString.WriteString(fmt.Sprintf("[%d] %s\n", shiftedIndex, file.Name()))
 		}
+		promptString.WriteString(fmt.Sprintln("Press ENTER to select the first option as default."))
 
-		answer, err := s.Prompt(fmt.Sprintf("Please specify the file to use as default CR in %s: %v\n", samplesDir, names))
-		defaultCR = filepath.Join(samplesDir, answer)
+		answer, err := s.Prompt(promptString.String())
 		if err != nil {
 			return nil, fmt.Errorf("could not obtain default CR from user prompt: %w", err)
 		}
-	} else {
-		// use only file in folder
-		defaultCR = filepath.Join(samplesDir, d[0].Name())
+		var parsedAnswer int
+		if answer == "" {
+			parsedAnswer = 1 // Default to the first choice
+		} else {
+			parsedAnswer, err = strconv.Atoi(answer)
+			if err != nil {
+				return nil, fmt.Errorf("could not obtain default CR from user prompt: %w", err)
+			}
+		}
+		fileName := files[parsedAnswer]
+		if len(fileName) == 0 {
```

Comment:
`files` is a map, when you get value by key, you can return two variable, the second indicate if the key exists.

```
fileName, exists := files[parsedAnswer]
if exists {
}